lib/plane.rb

```diff
@@ -0,0 +1,32 @@
+
+class Plane
+  DEFAULT_STATE = true
+  attr_reader :flying
+
+  def initialize
+     @flying = DEFAULT_STATE
+  end
+
+  def land(airport)
+    if landed
+      raise "Plane already landed"
+    end
```

Comment:
Tiny suggestion to improve readability :)

```suggestion
    raise "Plane already landed" if landed
```